api/tests/test_check_dem.py

```diff
@@ -0,0 +1,43 @@
+from pytest import raises
+from shapely.geometry import Polygon
+
+
+from hyp3_api.validation import GranuleValidationError, check_dem_coverage
+
+
+def test_dem_validation_intersections():
+    polygons = [
+        {  # Checks inland polygon
+            'polygon': Polygon(
+                [[-112.217865, 40.589935], [-111.80838, 38.969978], [-108.877068, 39.369526], [-109.213676, 40.98851],
+                 [-112.217865, 40.589935]]),
```

Comment:
could lower the precision of these, extra digits just distract, and might let you collapse the whole `Polygon()` call to one line

could also tailor the coordinates very specifically to the test cases (e.g. 0.001 away from coverage, 0.001 within coverage) instead of relying on the coordinates of a real granule